soc/arm64/qemu_cortex_a53/mmu_regions.c

```diff
@@ -19,6 +19,13 @@ static const struct arm_mmu_region mmu_regions[] = {
 			      DT_REG_ADDR_BY_IDX(DT_INST(0, arm_gic), 1),
 			      DT_REG_SIZE_BY_IDX(DT_INST(0, arm_gic), 1),
 			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_DEFAULT_SECURE_STATE),
+	
+	#ifdef CONFIG_PADDLE_LITE
```

Comment:
This module specific hardcoding is unlikely to be accepted

Reply:
Because deep learning model files cannot use fopen of the file system under QEMU. We directly modify the device tree to directly put the model into an address space and MMU.

Reply:
This is really not acceptable. If you need, create a `mmu_regions.c` file for your board.